idaes/core/base/property_set.py

```diff
@@ -947,6 +958,80 @@ class StandardPropertySet(PropertySetBase):
         units=pyunits.dimensionless,
     )
 
+    # Log10 terms
```

Comment:
We should probably think about whether we want to support log10 and ln separately, pick one of these as the standard, or just have "log" but let the user decide which base to use (which is probably a bad idea).

If we start supporting both log10 and ln then this list is going to get very long, plus we might want to ask if "log" should really be "ln". We should probably update the doc string for all the base-e terms to explicitly indicate the base (unless we go to a floating base).

Reply:
I think most people would expect log10, at least I would for these.  I could be wrong though, or it may depend on the property and use case.  At least I'd like to use log10 for what I'm working on now.  Not sure of the best approach though.

Reply:
For now I just put a few log10 properties in the ElectrolytePropertySet class, so we can figure this out later.